### PR TITLE
WSSQL-165 Do not attempt to parameterize prepared queries

### PR DESCRIPTION
**Pull request by rpastrana**
- Adds Treewalker member var to track option to attempt parameterization
- PrepareSQL logic opts out of parameterization attempt

Signed-off-by: Rodrigo Pastrana <rodrigo.pastrana@lexisnexis.com>

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).

## Checklist:
- [x] My code follows the code style of this project.
  - [x] My code does not create any new warnings from compiler, build system, or lint.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the HPCC Systems CONTRIBUTORS document (https://github.com/hpcc-systems/HPCC-Platform/wiki/Guide-for-contributors).
- [x] The change has been fully tested:
  - [x] I have performed unit tests to cover my changes.
  - [x] I have performed system test and covered possible regressions and side effects.
  - [x] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.

### CODE REVIEW

**Comment by rpastrana** [comment]
@Michael-Gardner please review.

File: esp/services/ws_sql/ws_sqlService.hpp [hunk 1]
```diff
@@ -121,7 +121,7 @@ class CwssqlEx : public Cwssql
 
     void fetchRequiredHpccFiles(IArrayOf<SQLTable> * sqltables);
     static void fetchRequiredHpccFiles(IArrayOf<SQLTable> * sqltables, HpccFiles * hpccfilecache);
-    HPCCSQLTreeWalker * parseSQL(IEspContext &context, StringBuffer & sqltext);
+    HPCCSQLTreeWalker * parseSQL(IEspContext &context, StringBuffer & sqltext, bool attemptParameterization);
```
Comment (Michael-Gardner) [inline esp/services/ws_sql/ws_sqlService.hpp]: Should we be stating the default value for attempt parameterization here?

Comment (rpastrana) [inline esp/services/ws_sql/ws_sqlService.hpp]: nope, moving to the function declaration

**Comment by rpastrana** [comment]
@Michael-Gardner please review latest change.

**Comment by Michael-Gardner** [comment]
@rpastrana Looks good

**Pull request merged**